WordPress/wp-feature-notifications
# Fix .nvmrc file

## What?


Recreate .nvmrc file due to encoding issues. I've gone with lts/gallium for now, as the build seems to work fine with this, however we can revisit the exact version down the line.

## Why?


See #166 

## How?


- Delete old .nvmrc file entirely
- Create new .nvmrc file

## Testing Instructions





- Use nvm to set correct node version
- Check for successful build

## Screenshots or screencast
